subprojects/performance/src/performanceTest/groovy/org/gradle/performance/experiment/java/GradleFeaturePreviewsPerformanceTest.groovy

```diff
@@ -31,12 +31,12 @@ class GradleFeaturePreviewsPerformanceTest extends AbstractCrossBuildPerformance
         runner.testGroup = "feature previews"
         runner.buildSpec {
             projectName(EXCLUDE_RULE_MERGING_TEST_PROJECT).displayName("advanced-pom-support").invocation {
-                tasksToRun("resolveDependencies").args("-Dorg.gradle.advancedpomsupport=true", '-PimprovedPomSupport=true').gradleOpts("-Xms${memory}", "-Xmx${memory}")
+                tasksToRun("resolveDependencies").args('-PimprovedPomSupport=true').gradleOpts("-Xms${memory}", "-Xmx${memory}")
```

Comment:
@melix These are the only changes I have doubt. It will make the performance project no longer use advanced pom support if run with a version before 4.6. And I could not find where the baseline version is determined.